client/src/components/Navbar.vue

```diff
@@ -44,10 +44,14 @@
       </div>
 
       <!-- A logged in Admin -->
-      <li v-if="isLoggedAdmin">
-        <a class="navbar-brand" href="/adminRegistrationMenu">Registration Menu</a>
-      </li>
-
+      <div v-if="isLoggedAdmin">
+        <li>
+          <a class="navbar-brand" href="/adminRegistrationMenu">Registration Menu</a>
+        </li>
+        <li v-if="isLoggedAdmin">
```

Comment:
why are we checking for isLoggedAdmin again, if the div will only display if the admin is logged in